cognite_toolkit/_cdf_tk/prototypes/interactive_init.py

```diff
@@ -166,64 +174,67 @@ def interactive(
 
         print(f"  [{'yellow' if mode == 'overwrite' else 'green'}]Using directory [bold]{init_dir}[/]")
 
+        selected: dict[str, dict[str, Any]] = {}
+        if package:
+            if not available.get(package):
+                raise ToolkitRequiredValueError(
+                    f"Package {package} is not known. Available packages are {', '.join(available.keys())}"
+                )
+
+            selected[package] = available[package].get("modules", {}).keys()
+
         loop = True
         while loop:
-            if not arg_selected:
-                if numeric:
-                    package_id = questionary.rawselect(
-                        "Which package would you like to include?",
-                        instruction="Type the number of your choice and press enter",
-                        choices=[questionary.Choice(value.get("title", key), key) for key, value in available.items()],
-                        pointer=POINTER,
-                        style=custom_style_fancy,
-                    ).ask()
-
-                else:
-                    package_id = questionary.select(
-                        "Which package would you like to include?",
-                        instruction="Use arrow up/down and ⮐  to save",
-                        choices=[questionary.Choice(value.get("title", key), key) for key, value in available.items()],
-                        pointer=POINTER,
-                        style=custom_style_fancy,
-                    ).ask()
+            if len(selected) > 0:
```

Comment:
I think this should not be there as I seem to be stuck in a loop if it is there.
```suggestion
```